src/mudlet.cpp

```diff
@@ -2279,22 +2279,22 @@ void mudlet::show_options_dialog()
 
 void mudlet::show_help_dialog()
 {
-    QDesktopServices::openUrl(QUrl("http://wiki.mudlet.org/w/Manual:Contents"));
+    QDesktopServices::openUrl(QUrl("https://wiki.mudlet.org/w/Manual:Contents"));
 }
 
 void mudlet::slot_show_help_dialog_video()
 {
-    QDesktopServices::openUrl(QUrl("http://www.mudlet.org/media/"));
+    QDesktopServices::openUrl(QUrl("https://www.mudlet.org/media/"));
 }
 
 void mudlet::slot_show_help_dialog_forum()
 {
-    QDesktopServices::openUrl(QUrl("http://forums.mudlet.org/"));
+    QDesktopServices::openUrl(QUrl("https://forums.mudlet.org/"));
 }
 
 void mudlet::slot_show_help_dialog_irc()
 {
-    QDesktopServices::openUrl(QUrl("http://webchat.freenode.net/?channels=mudlet"));
+    QDesktopServices::openUrl(QUrl("https://webchat.freenode.net/?channels=mudlet"));
```

Comment:
People might not spot this in current code but this is a __Web-based__ connection to the Mudlet IRC channel which provides an alternative to the (recently enhanced) built-in IRC sub-system...